src/app/modules/form/public-form/public-form.controller.ts

```diff
@@ -321,6 +328,56 @@ export const handleGetPublicForm: ControllerHandler<
           }
           return res.json({ form: publicForm, isIntranetUser })
         })
+    case FormAuthType.SGID_MyInfo: {
+      const accessTokenCookie = req.cookies[SGID_MYINFO_COOKIE_NAME]
+      if (!accessTokenCookie) {
+        return res.json({
+          form: publicForm,
+          isIntranetUser,
+        })
+      }
+      res.clearCookie(SGID_MYINFO_COOKIE_NAME)
+      res.clearCookie(MYINFO_LOGIN_COOKIE_NAME)
```

Comment:
After the revised impl (which separates the sgid cookie into two - sgid and sgid_myinfo), are we still using myinfo_login_cookie for sgid_myinfo?

Reply:
Yup we still do! This allows us to reuse code from MyInfo.

When SGID MyInfo mode is used, we need to also set the MyInfo cookie as we need somewhere to store the singpass username, and also because we reuse MyInfo functions, which read and depend on that cookie.